src/pykoala/data_container.py

```diff
@@ -39,7 +277,7 @@ def __init__(self, **kwargs):
         self.info = kwargs.get("info", dict())
         if self.info is None:
             self.info = dict()
-        self.log = kwargs.get("log", dict(corrections=dict()))
+        self.log = kwargs.get("log", dict())
         if self.log is None:
             self.log = dict()
```

Comment:
`self.log` cannot be `None`.
Why is it a dictionary, rather than a `HistoryLog`?

Reply:
Yes, once the class is properly implemented I will then include the changes on the rest of the classes (this is probably a change I made just after creating the branch).